src/App.tsx

```diff
@@ -1,24 +1,159 @@
-/* eslint-disable max-len */
-/* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
+import { addTodo, deleteTodo, getTodos } from './api/todos';
 import { UserWarning } from './UserWarning';
+import { TodoType } from './types/TodoType';
+import { Header } from './components/Header';
+import { Footer, SortType } from './components/Footer';
+import { Section } from './components/Section';
 
-const USER_ID = 0;
+const USER_ID = 6526;
+
+const getReorderedList = (sortType: SortType, list: TodoType[]) => {
+  switch (sortType) {
+    case SortType.Active:
+      return list.filter(item => !item.completed);
+    case SortType.Completed:
+      return list.filter(item => item.completed);
+    default:
+      return list;
+  }
+};
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+  const [todos, setTodos] = useState<TodoType[]>([]);
+  const [selectedSortType, setSortType] = useState(SortType.All);
+  const [isError, setError] = useState(false);
+  const [errorText, setErrorText] = useState('');
```

Comment:
you can leave just one error state variable
```suggestion
  const [errorText, setErrorText] = useState('');
```